python/ray/experimental/state.py

```diff
@@ -402,62 +411,75 @@ def micros(ts):
-        parent_trace["ph"] = "s"
-        parent_trace["name"] = "SubmitTask"
-        parent_trace["args"] = {}
-        parent_trace["id"] = str(worker)
+        parent_times = self._get_times(parent_info)
+        parent_trace = {
+            "cat": "submit_task",
+            "pid": "Node " + str(parent_worker["node_ip_address"]),
+            "tid": parent_info["worker_id"],
+            "ts": micros(min(parent_times)),
+            "ph": "s",
+            "name": "SubmitTask",
+            "args": {},
+            "id": str(worker)
+        }
         full_trace.append(parent_trace)
 
-        parent = dict()
-        parent["cat"] = "submit_task"
-        parent["pid"] = "Node " + str(parent_worker["node_ip_address"])
-        parent["tid"] = parent_info["worker_id"]
-        parent["ts"] = micros(min(parent_times))
-        parent["ph"] = "s"
-        parent["name"] = "SubmitTask"
-        parent["args"] = {}
-        parent["id"] = str(worker)
+        parent = {
+            "cat": "submit_task",
+            "pid": "Node " + str(parent_worker["node_ip_address"]),
+            "tid": parent_info["worker_id"],
+            "ts": micros(min(parent_times)),
+            "ph": "s",
+            "name": "SubmitTask",
+            "args": {},
+            "id": str(worker)
+        }
         full_trace.append(parent)
 
-      task_trace = dict()
-      task_trace["cat"] = "submit_task"
-      task_trace["pid"] = "Node " + str(worker["node_ip_address"])
-      task_trace["tid"] = info["worker_id"]
-      task_trace["ts"] = micros(min(times))
-      task_trace["ph"] = "f"
-      task_trace["name"] = "SubmitTask"
-      task_trace["args"] = {}
-      task_trace["id"] = str(worker)
+      task_trace = {
+          "cat": "submit_task",
+          "pid": "Node " + str(worker["node_ip_address"]),
+          "tid": info["worker_id"],
+          "ts": micros(min(times)),
+          "ph": "f",
+          "name": "SubmitTask",
+          "args": {},
+          "id": str(worker)
+      }
       full_trace.append(task_trace)
 
-      task = dict()
-      task["name"] = info["function_name"]
-      task["cat"] = "ray_task"
-      task["ph"] = "X"
-      task["ts"] = micros(min(times))
-      task["dur"] = micros(max(times)) - micros(min(times))
-      task["pid"] = "Node " + str(worker["node_ip_address"])
-      task["tid"] = info["worker_id"]
-      task["args"] = info
+      task = {
+          "name": info["function_name"],
+          "cat": "ray_task",
+          "ph": "X",
+          "ts": micros(min(times)),
+          "dur": micros(max(times)) - micros(min(times)),
+          "pid": "Node " + str(worker["node_ip_address"]),
+          "tid": info["worker_id"],
+          "args": info
+      }
       full_trace.append(task)
 
-    with open(path, 'w') as outfile:
+    with open(path, "w") as outfile:
       json.dump(full_trace, outfile)
     task_info
 
-  def get_times(self, data):
+  def _get_times(self, data):
```

Comment:
Methods that are not part of the API should be hidden from users (via a leading underscore). Note that this prevents the method from being shown by IPython's tab completion.